astropy/coordinates/baseframe.py

```diff
@@ -716,6 +716,11 @@ def get_frame_attr_names(cls):
         return {name: getattr(cls, name)
                 for name in cls.frame_attributes}
 
+    @property
+    def _frame_attr_dict(self):
```

Comment:
My sense would be not to add this new property, given that it is only used in one place, and really baseframe has too many of these already...